tic-tac-toe/src/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import * as serviceWorker from './serviceWorker';
+
+function Square({ value, onClick }) {
+
+  return (
+    <button className="square" onClick={onClick}>
+      {value}
+    </button>
+  );
+}
+
+ReactDOM.render(<Square />, document.getElementById("root"));
```

Comment:
This line can be removed. 

The `<Game/>` component already renders the instances of the `<Square/>`.

Also, [ReactDOM.render() docs](https://reactjs.org/docs/react-dom.html#reference) states `If the React element was previously rendered into container, this will perform an update on it and only mutate the DOM as necessary to reflect the latest React element.` 

This means that  `ReactDOM.render()` is updated by the `ReactDOM.render()` on line 126. 

If this line remained without line 126, then the app would render only one instance of the `<Square/>` component.